--- generate-xform_new.js	2023-10-26 21:18:12.796185727 -0500
***************
*** 6,13 ****
  const path = require('path');
  const htmlParser = require('node-html-parser');
  const logger = require('../logger');
! const db = require('../db');
! const formsService = require('./forms');
  const markdown = require('../enketo-transformer/markdown');
  
  const MODEL_ROOT_OPEN = '<root xmlns="http://www.w3.org/2002/xforms" xmlns:xf="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema">';
--- 6,13 ----
  const path = require('path');
  const htmlParser = require('node-html-parser');
  const logger = require('../logger');
! // const db = require('../db');
! // const formsService = require('./forms');
  const markdown = require('../enketo-transformer/markdown');
  
  const MODEL_ROOT_OPEN = '<root xmlns="http://www.w3.org/2002/xforms" xmlns:xf="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema">';
***************
*** 15,22 ****
  const JAVAROSA_SRC = / src="jr:\/\//gi;
  const MEDIA_SRC_ATTR = ' data-media-src="';
  
! const FORM_STYLESHEET = path.join(__dirname, '../xsl/openrosa2html5form.xsl');
! const MODEL_STYLESHEET = path.join(__dirname, '../enketo-transformer/xsl/openrosa2xmlmodel.xsl');
  const XSLTPROC_CMD = 'xsltproc';
  
  const processErrorHandler = (xsltproc, err, reject) => {
--- 15,23 ----
  const JAVAROSA_SRC = / src="jr:\/\//gi;
  const MEDIA_SRC_ATTR = ' data-media-src="';
  
! // const FORM_STYLESHEET = path.join(__dirname, '../xsl/openrosa2html5form.xsl');
! // const MODEL_STYLESHEET = path.join(__dirname, '../enketo-transformer/xsl/openrosa2xmlmodel.xsl');
! const { FORM_STYLESHEET, MODEL_STYLESHEET } = require('../xsl/xsl-paths');
  const XSLTPROC_CMD = 'xsltproc';
  
  const processErrorHandler = (xsltproc, err, reject) => {
